yarn.lock

```diff
@@ -11413,6 +11444,40 @@ __metadata:
   languageName: node
   linkType: hard
 
+"jsdom@npm:^23.2.0":
```

Comment:
It's unfortunate that this pulls in a version of `jsdom` that's effectively unused; this leaves us with an extra 4MB of dependencies to ship around. I'm wondering if we could resolve this with Yarn resolutions: https://yarnpkg.com/configuration/manifest#resolutions

I'm happy to try out this change and push it to your branch if that's OK with you.

Reply:
I'm good with that.
I'll fix the first two issues Tuesday or Wednesday.